app/src/main/java/io/intrepid/contest/ContestApplication.java

```diff
@@ -4,14 +4,15 @@
 
 import io.intrepid.contest.base.PresenterConfiguration;
 import io.intrepid.contest.rest.RetrofitClient;
+import timber.log.Timber;
 import uk.co.chrisjenx.calligraphy.CalligraphyConfig;
 
 public class ContestApplication extends Application {
 
     @Override
     public void onCreate() {
         super.onCreate();
-
+        Timber.plant(new Timber.DebugTree());
```

Comment:
I think if you plant debug tree, it wont show logs in release version. I'm not sure, need to check in with @jmccormack200